cirq/ops/pauli_string.py

```diff
@@ -697,8 +723,8 @@ def __rpow__(self, base):
                 exponent_pos=-half_turns / 2)
         return NotImplemented
 
-    def map_qubits(self, qubit_map: Dict[raw_types.Qid, raw_types.Qid]
-                   ) -> 'PauliString':
+    def map_qubits(self,
+                   qubit_map: Dict[TKey, TKey2]) -> 'cirq.PauliString[TKey2]':
```

Comment:
Why is it Tkey -> Tkey2? Can't it be Tkey -> Tkey?

Reply:
The input type is the dictionary key type, the output type is the dictionary value type. This is the most precise way of specifying what the method does. Using a union would give less type information.

Reply:
Got it, makes sense. Thanks!